.github/workflows/ps-ce-sdk-psgallery-publish.yaml

```diff
@@ -0,0 +1,26 @@
+# **************************************************************************
+#  Copyright (c) Cloud Native Foundation.
+#  SPDX-License-Identifier: Apache-2.0
+# **************************************************************************
+
+name: Publish to PSGallery
+
+on:
+  create:
+    tags:
+      - v*
+
+jobs:
+  psgallery-publish:
+    name: Publish CloudEvents.Sdk Module to PSGallery
+    runs-on: "windows-latest"
+    
+    timeout-minutes: 10
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Build Test and Publish
+        shell: pwsh
+        run: ./publish.ps1 -NuGetApiKey ${{ secrets.CLOUDEVENTS_SDK_PUBLISHER_API_KEY }}
```

Comment:
In addition, IMHO we should also create a Github release and update the CHANGELOG as part of this workflow.

Reply:
Renamed the workflow to 'release' and perform the below steps:
1. Build and Run all Tests
2. Publish to PSGallery
3. Create CHANGELOG
4. Create GitHub release